tests/integration/targets/sysrc/main/main.yml

```
@@ -0,0 +1,223 @@
+---
+- name: Test on FreeBSD VMs
+  when:
+    - ansible_facts.virtualization_type != 'docker'
+    - ansible_facts.distribution == 'FreeBSD'
+  block:
+    ##
+    ## sysrc - /etc/rc.conf manipulation
+    ##
+    - name: add to /etc/rc.conf
```

```suggestion
    - name: Add to /etc/rc.conf
```
1) would be good to have tests corresponding to the documentation demands for `EXAMPLES`
2) and to cover `check_mode`